tests/4.5/taskloop/test_taskloop_num_tasks.c

```diff
@@ -0,0 +1,76 @@
+//------------------------------------------- test_taskloop_num_tasks.c --------------------------------------//
+//
+// OpenMP API Version 4.5 September 2015
+//
+// This test checks the 'taskloop' directive with the 'num_tasks' clause specified.
+// The 'taskloop' construct parallelizes loops with independent iterations by creating tasks. 
+// It allows for efficient parallel execution of loop iterations by distributing them among multiple threads. 
+// The 'num_tasks' clause variable ensures that the loop iterations are shared amoung created 'num_tasks'. 
+// i.e the loop iterations are divided amoung number of tasks (each num_task group is run by same the thread).   
+//----------------------------------------------------------------------------------------------------------//
+
+#include <stdio.h>
+#include <omp.h>
+#include "ompvv.h"
+
+#define NUM_THREADS 100
```

Comment:
Why do we need 100 threads ?

Reply:
Just for testing purpose considered 100 threads.